src/components/NavBar/index.js

```diff
@@ -1,12 +1,12 @@
-import React from "react";
+import React, { useState } from "react";
 import { useDispatch } from "react-redux";
+import { Menu, Button } from "antd";
+import "antd/dist/antd.css";
```

Comment:
you can import this in main index.js file at the root in src because it's gonna become globally available